app/bundles/PluginBundle/Controller/PluginController.php

```diff
@@ -440,32 +410,25 @@ public function reloadAction()
                     $pluginMetadata[$bundleName] = [];
                 }
                 $pluginMetadata[$bundleName][$meta->getName()] = $meta;
-
-                $table = $meta->getTableName();
-
+                $table                                         = $meta->getTableName();
```

Comment:
This line could be moved below the `if` on line 414. We aren't using the `$table` var until after that `if`, and the code has better readability without the huge space to line up the equals.